static_src/constants.js

```diff
@@ -2,15 +2,32 @@
 import keymirror from 'keymirror';
 
 const appStates = {
+  crashed: 'CRASHED',
+  down: 'DOWN',
+  flapping: 'FLAPPING',
```

Comment:
I got this from the CLI, but not sure under what conditions all of these might appear.